packaging/deb/rules

```diff
@@ -53,6 +53,8 @@ override_dh_auto_install:
 	install -m 644 ood-portal-generator/share/maintenance.html $(APACHE_DIR)/public/maintenance/index.html
 	install -m 644 ood-portal-generator/share/need_auth.html $(APACHE_DIR)/public/need_auth.html
 	install -m 644 hooks/hook.env.example $(CONFIG_DIR)/hook.env
+	mkdir -p $(DESTDIR)/opt/ood/ondemand
+	install -m 644 packaging/files/profile.deb $(DESTDIR)/opt/ood/ondemand/enable
```

Comment:
Shouldn't we keep the profile name? Or this is just for Ubuntu? 

I guess I'm worried about the toggle in the documentation. Do this for X and this for Y. 

I guess I just assumed what you'd thought was a singe `/opt/ood/profile` that both nginx stage and ood-portal-generator source and that folks can override in /etc/ood/profile.

Reply:
I don't think we need to document this file, we don't document the same file for SCL platforms.  This is a file that if modified, package updates will overwrite.  If you'd prefer I can move it to `/opt/ood/profile`, the main motivation for this path was to have Ubuntu use same file as EL platforms which use SCL.

Reply:
We do have documentation for `/opt/ood/nginx_stage/etc/profile` which on Ubuntu would need to change to `/opt/ood/ondemand/enable`. Or should the docs point to /opt/ood/ondemand/enable instead of /opt/ood/nginx_stage/etc/profile?

https://github.com/OSC/ood-documentation/blob/0a5fe4babd2f60acd678d169b48b9b7958f21320/source/reference/pun-environment.rst?plain=1#L36

My concern is doing different things on different platforms can be confusing not only for folks to implement for us to explain.

Reply:
OK I think this conversation just illustrates how easily this can get confusing.

It seems like I need a minute to look this over in more detail and that the docs would still hold.